### PR TITLE
feat: add lifi-explorer to main-menu

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-5103

image

### CODE REVIEW

**Comment by dennyscode** [comment]
There´s a few open questions in the ticket

**Review by Addminus — CHANGES_REQUESTED** [review]
it makes more sense to add it to the top level of the menu. its not just for devs
image
also it should be `LIFI-Explorer`

**Comment by dennyscode** [comment]
@Addminus 

1.) Added to top-menu
2.) Renamed to "LI.FI Explorer" (with dot, without "-")

**Comment by chybisov** [comment]
> @Addminus 
> 
> 
> 
> 1.) Added to top-menu
> 
> 2.) Renamed to "LI.FI Explorer" (with dot, without "-")

Thanks for the second 😅 Came here to say the same 😄

**Comment by dennyscode** [comment]
@chybisov Thanks for checking in and confirming that!! 👍

**Review by chybisov — CHANGES_REQUESTED** [review]

File: packages/dapp/src/i18n/en/translation.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "navbarMenu": {
       "theme": "Theme",
       "developers": "Developers",
+      "lifi_explorer": "LI.FI Explorer",
```
[the remark this answers is not in the document]
Comment (dennyscode) [reply packages/dapp/src/i18n/en/translation.json:24]: Of course! Thank you, not sure what made me use that snake casing here again 🐍

**Review by chybisov — APPROVED** [review]

**Pull request merged**